src/api/MoneyMoovApiClient.ts

```diff
@@ -0,0 +1,73 @@
+import { deletePaymentRequest, getPaymentRequests, voidPaymentRequest } from "./PaymentRequestClient";
+import { PaymentRequestPageResponse } from "./types/ApiResponses";
+import { ApiError } from "./types/ApiResponses";
+
+class MoneyMoovApiClient {
+  apiUrl: string;
+  token: string;
+
+  /**
+   * @param apiUrl The api base url
+   * @param token The token
+   */
+  constructor(apiUrl: string, token: string) {
+    this.apiUrl = apiUrl;
+    this.token = token;
+  }
+
+  /**
+   * Gets a paged list of Payment Requests
+   * @param apiUrl The base api url. E.g https://api.nofrixion.com/api/v1
+   * @param token The auth token.
+   * @param page The first page. Default is 1
+   * @param size The page size. Default is 20
+   * @returns A PaymentRequestPageResponse if successful. An ApiError if not successful.
+   */
+  getPaymentRequests = async (
+    apiUrl: string,
+    token: string,
+    page: number = 1,
+    size: number = 20
+  ): Promise<{
+    data?: PaymentRequestPageResponse | undefined;
+    error?: ApiError | undefined;
+  }> => {
+    return await getPaymentRequests(apiUrl, token, page, size);
+  };

```

Comment:
Not sure if we need to pass the `apiUrl` and `token` as a param. Couldn't we use `this.apiUrl` here?

 I think both values should be accessible since it's a method within the same class (`MoneyMoovApiClient`).